src/route53.coffee

```diff
@@ -114,7 +114,13 @@ module.exports = (config) ->
     # We need to determine if the requested hostname is currently assigned in a DNS record.
     {current_ip_address, current_type} = yield @get_current_record(record.hostname, record.zone_id)
 
-    if current_ip_address?
+    if current_ip_address == record.ip_address and
```

Comment:
This is very minor point, but Dan has mentioned that he and Matthew prefer to see `&&` and  `||` instead of `and` and `or`.  It's not a super big deal, but as you code on stuff directly reviewed by Dan (particularly fairmont) he may mention it to you.